frontend/packages/core-api-client/src/lib/client/Recipient.ts

```diff
@@ -0,0 +1,37 @@
+import { FormLookup, RecordLookup } from '../types';
+import {
+  Recipient,
+  RecipientDefinition,
+  RecipientList,
+} from '../types/client/Recipient';
+
+import { RecordClient } from './Record';
+
+export class RecipientClient {
+  recordClient: RecordClient;
+
+  constructor(recordClient: RecordClient) {
+    this.recordClient = recordClient;
+  }
+
+  create = (recipient: RecipientDefinition): Promise<Recipient> => {
+    return Promise.resolve({
+      id: 'id',
+      ...recipient,
+    });
+  };
+
+  list = (args: FormLookup): Promise<RecipientList> => {
+    return Promise.resolve({ items: [] });
```

Comment:
I don't see the use of the `items` key. Could this just be an array?

Reply:
it seems to be the current standard of what the api returns. if we extract the items at some point, this could be an array, but if we ever want to do pagination, this might be necessary

Reply:
yeah good shout, this is better for pagination